### PR TITLE
Changed root package to `org.locationtech.rasterframes`

### PR DESCRIPTION
**Pull request by metasim**
from `astraea.spark.rasterframes` to `org.locationtech.rasterframes`

### CODE REVIEW

File: docs/src/main/tut/release-notes.md [hunk 1]
```diff
@@ -113,7 +114,7 @@
 ### 0.5.9
 
 * Ported to sbt 1.0.3
-* Added sbt-generated `astraea.spark.rasterframes.RFBuildInfo`
```
Comment (vpipkt) [inline docs/src/main/tut/release-notes.md]: should not change this one, I would think.

File: docs/src/main/tut/release-notes.md [hunk 2]
```diff
@@ -18,7 +19,7 @@
 * _Breaking_: renamed `agg_histogram` to `agg_approx_histogram`, `local_agg_stats` to `agg_local_stats`, `local_agg_max` to `agg_local_max`, `local_agg_min` to `agg_local_min`, `local_agg_mean` to `agg_local_mean`, `local_agg_data_cells` to `agg_local_data_cells`, `local_agg_no_data_cells` to `agg_local_no_data_cells`.
 * _Breaking_: `CellHistogram` no longer carries along approximate statistics, due to confusing behavior. Use `agg_stats` instead.
 * Introduced `LocalCellStatistics` class to wrap together results from `LocalStatsAggregate`.
-* _Breaking_: `TileDimensions` moved from `astraea.spark.rasterframes` to `astraea.spark.rasterframes.model`.
+* _Breaking_: `TileDimensions` moved from `org.locationtech.rasterframes` to `org.locationtech.rasterframes.model`.
```
Comment (vpipkt) [inline docs/src/main/tut/release-notes.md]: These should be termed as "moved from astraea.spark.rasterframes to org.locationtech.rasterframes.model."

File: docs/src/main/tut/release-notes.md [hunk 3]
```diff
@@ -52,9 +53,9 @@
   tile image directory structure and Leaflet/OpenMaps-enabled HTML file. 
 * Added [experimental DataSource implementations](https://github.com/locationtech/rasterframes/tree/develop/experimental/src/main/scala/astraea/spark/rasterframes/experimental/datasource/awspds) for [MODIS](https://registry.opendata.aws/modis/) and [Landsat 8](https://registry.opendata.aws/landsat-8/) catalogs on AWS PDS.   
 * _Change_: Default interpoation for `toRaster` and `toMultibandRaster` has been changed from `Bilinear` to `NearestNeighbor`.
-* _Breaking_: Renamed/moved `astraea.spark.rasterframes.functions.CellStatsAggregateFunction.Statistics` to
-`astraea.spark.rasterframes.stats.CellStatistics`.
-* _Breaking_: `HistogramAggregateFunction` now generates the new type `astraea.spark.rasterframes.stats.CellHistogram`.
+* _Breaking_: Renamed/moved `org.locationtech.rasterframes.functions.CellStatsAggregateFunction.Statistics` to
```
Comment (vpipkt) [inline docs/src/main/tut/release-notes.md]: should not change these lines.

File: datasource/src/test/scala/org/locationtech/rasterframes/datasource/geotrellis/GeoTrellisDataSourceSpec.scala [hunk 4]
```diff
@@ -1,7 +1,7 @@
 /*
  * This software is licensed under the Apache 2 license, quoted below.
  *
- * Copyright 2017-2018 Azavea & Astraea, Inc.
```
Comment (vpipkt) [inline datasource/src/test/scala/org/locationtech/rasterframes/datasource/geotrellis/GeoTrellisDataSourceSpec.scala:4]: ? Not sure about this change of copyright as it removes Azavea.

File: core/src/main/scala/org/locationtech/rasterframes/ref/HttpRangeReader.scala [hunk 5]
```diff
@@ -1,26 +1,30 @@
 /*
- * Copyright 2016 Azavea
```
Comment (vpipkt) [inline core/src/main/scala/org/locationtech/rasterframes/ref/HttpRangeReader.scala]: again change of copyright owner?

File: datasource/src/main/scala/org/locationtech/rasterframes/datasource/geotrellis/GeoTrellisRelation.scala [hunk 6]
```diff
@@ -1,7 +1,7 @@
 /*
  * This software is licensed under the Apache 2 license, quoted below.
  *
- * Copyright 2017-2018 Azavea & Astraea, Inc.
```
Comment (vpipkt) [inline datasource/src/main/scala/org/locationtech/rasterframes/datasource/geotrellis/GeoTrellisRelation.scala:4]: change of owner

File: core/src/main/scala/org/locationtech/rasterframes/extensions/Implicits.scala [hunk 7]
```diff
@@ -61,17 +63,17 @@ trait Implicits {
     val self: RDD[(SpaceTimeKey, T)] with Metadata[TileLayerMetadata[SpaceTimeKey]]
   )(implicit spark: SparkSession) extends SpatioTemporalContextRDDMethods[T]
 
-  private[astraea]
+  private[rasterframes]
```
Comment (vpipkt) [inline core/src/main/scala/org/locationtech/rasterframes/extensions/Implicits.scala:66]: Interesting but does not change the scope, correct?

**Review by vpipkt — CHANGES_REQUESTED** [review]
One thing I noticed in several places was the year of the copyright has been changed everywhere to 2019. I am not sure if that is proper.

There are some places where we are changing the copyright author, and those would need to be considered. I tried to comment on all of them inline. It seems that should not be changed.

File: core/src/main/scala/org/locationtech/rasterframes/extensions/Implicits.scala (hunk 7 above)
Comment (metasim) [reply core/src/main/scala/org/locationtech/rasterframes/extensions/Implicits.scala:66]: It was `astraea` before so I could access them from our other code.... but that was a sneaky hack. So this does technically change the scope, but only in a way that forces me to "do the right thing" elsewhere.

**Comment by metasim** [comment]
> One thing I noticed in several places was the year of the copyright has been changed everywhere to 2019. I am not sure if that is proper.
> 
> There are some places where we are changing the copyright author, and those would need to be considered. I tried to comment on all of them inline. It seems that should not be changed.

Good catch. That was something IntelliJ unfortunately did during the refactoring process. Wish it didn't. I'll go back and see if there's an easy way to try to fix that. Not sure how much it matters. :/

File: core/src/main/scala/org/locationtech/rasterframes/extensions/Implicits.scala (hunk 7 above)
Comment (vpipkt) [reply core/src/main/scala/org/locationtech/rasterframes/extensions/Implicits.scala:66]: 👍

**Pull request merged**